simplified-main/src/main/java/org/nypl/simplified/main/MainServices.kt

```diff
@@ -943,6 +945,12 @@ internal object MainServices {
       serviceConstructor = { return@addService AudioBookManifests }
     )
 
+    addService(
+      message = strings.bootingGeneral("metrics service"),
+      interfaceType = MetricServiceType::class.java,
+      serviceConstructor = { MetricService(context) }
+    )
+
```

Comment:
This makes the metrics service mandatory for all builds. We should really be looking up an implementation via `ServiceLoader` and adding it optionally.